lib/cisco_node_utils/syslog_settings.rb

```diff
@@ -0,0 +1,81 @@
+# Syslog Settings provider class
+#
+# Jonathan Tripathy et al., September 2015
+#
+# Copyright (c) 2014-2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require File.join(File.dirname(__FILE__), 'node')
+require File.join(File.dirname(__FILE__), 'interface')
+
+module Cisco
+  # NtpServer - node utility class for NTP Server configuration management
+  class SyslogSettings < NodeUtil
+    attr_reader :timestamp
+
+    def initialize(syslogsettings_name,
+                   syslogsettings_timestamp,
```

Comment:
`s/syslogsettings_//g`

The prefix is unnecessary and makes the code harder to read. Plus it requires you to manually override the attr_reader implementations below - if you just change these to 'name' and 'timestamp' then it's a win-win-win.